### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by bkuhl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,64 @@
...
+* [Continuous Integration/Deployment Workflow Diagram](http://realpage.github.io/lumen-starter/ci-cd-foundation-lumen-starter-workflow.png)
+* [Roadmap](#roadmap)
+* [FAQ](#faq)
+
+<a name="whats-included" />
 ### What's Included
 
  * Latest version of Laravel
- * Pre-configured `docker-compose.yml` that uses nginx, php-fpm and PostgreSQL
+ * Pre-configured `docker-compose.yml` that uses nginx, php-fpm and PostgreSQL ([How can I use MySQL?](#use-mysql))
```
Comment (jlaswell) [inline README.md]: Need a `.` after PostgreSQL.

**Comment by jlaswell** [comment]
Quick update and this LGTM! Nice work

File: README.md (hunk 1 above)
Comment (bkuhl) [inline README.md]: None of the other bullets have a `.`, do you still think this should?

Comment (jlaswell) [inline README.md]: It's a full sentence, so I would think so. You could put the link as a sub-bullet if you'd prefer consistency

File: README.md [hunk 2]
```diff
@@ -1,27 +1,64 @@
-## Laravel Starter Template [![Build Status](https://travis-ci.org/realpage/laravel.svg?branch=master)](https://travis-ci.org/realpage/laravel)
+## Laravel Starter Template [![Build Status](https://travis-ci.org/realpage/laravel-starter.svg?branch=master)](https://travis-ci.org/realpage/laravel-starter)
 
-This [Laravel](https://www.laravel.com) starter template is intended to be forked and used for new projects.
+This [Laravel](https://www.laravel.com) starter template is intended to be copied and used for new projects.
```
Comment (millidavids) [inline README.md]: cloned?

File: README.md (hunk 1 above)
Comment (bkuhl) [inline README.md]: Updated

File: README.md (hunk 2 above)
Comment (bkuhl) [inline README.md]: Hrmm... I'll remove those "instructions" as they should really look directly at another section of the readme for that

**Comment by bkuhl** [comment]
Requested changes are in based on feedback, John gave his LGTM pending those changes so I'll go ahead and merge.

**Pull request merged**